src/VisualStudio/IntegrationTest/TestUtilities/InProcess/VisualStudioWorkspace_InProc.cs

```diff
@@ -30,7 +30,17 @@ private VisualStudioWorkspace_InProc()
         }
 
         public static VisualStudioWorkspace_InProc Create()
-            => new VisualStudioWorkspace_InProc();
+        {
+            // TOOD: revert try-catch added for non-serializable CompositionFailedException diagnosis.
```

Comment:
>TOOD [](start = 15, length = 4)

Should this be done in this PR or in a follow up (tracked by an issue)?

Reply:
I am going to revert this now. Integration tests are failing on the PR due to this, but this change did not help. Bit more head scratching to do integration test failures do not repro locally for me.